### ISSUE TITLE
The content

### ISSUE THREAD

**Issue by theIYD** [stach/functions.js#1]
First of all, i want to Thank you for the **Christmas Surprise**  to make an effort to build a documentation Github Page for functions.js. I hope you loved the small library. 

Talking about the content, we need a minimalistic, simple and clean documentation which does not simply confuse the user who is reading it. 

A sample heirarchy might be like below: 

-**Function name**
Here goes the function name with the actual implemented one & the one user is going to use it. 

-**Arguments** 
Here the argument names with their types will come. (Actually, in JSDocs of the functions, i have forgotten to define their types, although for an early production this might be empty.)

-**Description** 
Now this is the most **important**  part. The description must be more elaborated and more descriptive compared to the one liners in the Github Repo Methods Table. I know it will be difficult, but i will be helping you to build this together. I want the user to understand the funciton and let him go easy on this. 

-**Return type** 
Here goes the return types, what does the function return after it executes. 

-**Examples** 
Here, 2 to 3 unique examples of the function. 

So this might be the standard of each and every function. 

The homepage might be minimalistic and impressing to the user. Animations are a plus point. 

Please let me know a social contact so that i could talk to you more. 

Thanks again for the effort 😀😊 @stach.

**Comment by stach** [comment]
Yep, For the function list, it is available on https://stach.github.io/functions.js/functions.html

**Comment by stach** [comment]
The Data Flow :

I basically transformed all of your function list and upload them to firebase in a JSON format.
https://functionsjs-e4f26.firebaseio.com/rootRef/functions.json

**Comment by stach** [comment]
I'm actually thinking about a bulma like documentation. (https://bulma.io/documentation/overview/start/). Where docs are divided by categories (Math, Strings & Algorithms). What do you have in mind?

**Comment by theIYD** [comment]
The Data flow might be correct. But i need a great description related to each function. I guess i might make description JSON file and send it over to you to include those descriptions. But that would be the last choice. Let me know your thoughts on elaborating the description.

**Comment by stach** [comment]
How about : 

- Function Category
- Short description
- Example code
- What the function will return

> That is a simple data flow. 

But about making a great description for each one of em. I say I write a description for 50 functions. You're also going to write a description for 50 functions. What do you say?

**Comment by theIYD** [comment]
I will write the descriptions for all the functions in a new few days. 

Considering the above data flow, let the category go below the function name. Also, a table would not be the perfect UI to display all the functions. A **card-like** structure might do the work. 

Thanks @stach

**Comment by stach** [comment]
I know. All functions listed would be a really shit documentation. I agree with you. Anyway, is there a faster way to communicate instead of GitHub Issues? Twitter probably?

**Comment by theIYD** [comment]
Hit me up on Hangouts. (dargahidrees@gmail.com)

**Comment by stach** [comment]
You open hangouts on a regular basis? Not like messenger or others?

**Comment by theIYD** [comment]
I am always online on Hangouts because i have other friends and online developers too. You can hit me up on Hangouts. 

Also i have followed you on Twitter, but i do not use Twitter often.
